src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,26 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class HomePage extends BasePage {
+
+   WebDriver driver;
+  WebDriverWait wait;
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )

![image](https://user-images.githubusercontent.com/104761482/214539738-45377e11-a1d9-4396-8e76-d3f734ea237d.png)

![image](https://user-images.githubusercontent.com/104761482/214539784-9b7e6209-9dd6-41c1-a29c-883689bf0b17.png)